src/data/matrix/basic.lean

```diff
@@ -728,7 +728,7 @@ by { ext, apply neg_dot_product }
 lemma mul_vec_neg (v : n → α) (A : matrix m n α) : mul_vec A (-v) = - mul_vec A v :=
 by { ext, apply dot_product_neg }
 
-lemma smul_mul_vec_assoc (A : matrix n n α) (b : n → α) (a : α) :
+lemma smul_mul_vec_assoc (A : matrix m n α) (b : n → α) (a : α) :
   (a • A).mul_vec b = a • (A.mul_vec b) :=
 begin
```

Comment:
while you're here, this proof can change to `by { ext, apply smul_dot_product }`

Reply:
Will do this change as well. Thanks!